src/main/java/frc/robot/commands/HoldArmCommand.java

```diff
@@ -26,24 +32,27 @@ public HoldArmCommand(Arm robotArm) {
 
     @Override
     public void initialize() {
```

Comment:
How are you determining your static and gravity gains for this system?
https://www.reca.lc/arm